tests/clar.c

```diff
@@ -419,37 +423,33 @@ clar_test_run(void)
 {
 	if (_clar.argc > 1)
 		clar_parse_args(_clar.argc, _clar.argv);
-
 	if (!_clar.suites_ran) {
 		size_t i;
 		for (i = 0; i < _clar_suite_count; ++i)
 			clar_run_suite(&_clar_suites[i], NULL);
 	}
-
 	return _clar.total_errors;
 }
 
 void
-clar_test_shutdown(void)
+clar_test_shutdown()
 {
 	clar_print_shutdown(
 		_clar.tests_ran,
-		(int)_clar_suite_count,
-		_clar.total_errors
+		_clar.suites_ran,
+		_clar.total_errors,
+		_clar.total_skipped
 	);
 
 	clar_unsandbox();
 }
 
 int
-clar_test(int argc, char **argv)
+clar_test(void)
 {
 	int errors;
-
-	clar_test_init(argc, argv);
```

Comment:
Why is this being removed? Looks like it should stay here after all to parse our arguments. At least I couldn't see any other place where it's been added instead

Reply:
because this function was never used (see how things work in the main), mainly because we  added the tracing setup (some code coverage would have caught this). Removing this init here allowed us to use this function in the main.